### DISCUSSION TITLE
Getting ingredients in many-to-many relation

### DISCUSSION THREAD

**Discussion by kdurek** [prisma/prisma#12760] [Q&A] [answered]
```javascript
model Recipe {
  id          String       @id @default(cuid())
  name        String       @db.VarChar(255)
  content     String?
  ingredients Ingredient[]
  author      User         @relation(fields: [authorId], references: [id])
  authorId    String
  published   Boolean      @default(false)
  createdAt   DateTime     @default(now())
  updatedAt   DateTime     @updatedAt

  @@index(authorId)
}

model Ingredient {
  id      String   @id @default(cuid())
  name    String   @db.VarChar(255)
  recipes Recipe[]
}
```

Hey guys i spent too many hours wondering about this and gave up
I want to get ingredients that contains related recipe in recipes field

```javascript
prisma.ingredient.findMany({
  where: {
    recipes: {
      id: "recipeId",
    }
  },
});
```

Is it possible not using `prisma.recipe.findUnique`?

**Comment by pantharshit00** [comment, marked as answer]
Hello @kdurek, our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

I think you can use the following query here in order to get the related recepie:
```ts
  const data = await prisma.ingredient.findMany({
    where:{
      recipes:{
        some:{
          name:{
            contains:'name here'
          }
        }
      }
    }
  })
```

**Reply by vladi-stevanovic** [reply]
👋 Hello @kdurek did you perhaps have a chance to check Harshit's recommendation? 

Let us know if you still have any questions, or feel free to "Mark as answer" if this discussion is now solved!
It would be super useful for other users facing this problem! 🙌

**Reply by kdurek** [reply]
Hello, it was so long ago so I can't really tell if it's working but it seems legit :D

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
